client/routes/execution/history.vue

```diff
@@ -265,31 +302,15 @@ export default {
       setTimeout(() => this.scrollEventIntoView(this.eventId), 100);
     },
     setTsFormat(tsFormat) {
-      this.tsFormat = tsFormat
-      localStorage.setItem(`${this.$route.params.domain}:history-ts-col-format`, tsFormat)
+      this.tsFormat = tsFormat;
+      localStorage.setItem(`${this.domain}:history-ts-col-format`, tsFormat);
     },
     setCompactDetails(compact) {
       const { scrollerGrid } = this.$refs;
       this.compactDetails = compact;
-      localStorage.setItem(`${this.$route.params.domain}:history-compact-details`, JSON.stringify(compact));
+      localStorage.setItem(`${this.domain}:history-compact-details`, JSON.stringify(compact));
       scrollerGrid.forceUpdate();
-    },
-    timeCol(ts, i) {
-      if (i === -1) {
-        return '';
-      }
-
-      if (i === 0 || this.tsFormat !== 'elapsed') {
-        return ts.format('MMM Do h:mm:ss a')
-      }
-
-      let deltaFromPrev = moment.duration(ts - this.$parent.results[i - 1].timestamp),
-          elapsed = moment.duration(ts - this.$parent.results[0].timestamp).format()
-
-      if (deltaFromPrev.asSeconds() >= 1) {
-        elapsed += ` (+${deltaFromPrev.format()})`
-      }
-      return elapsed
```

Comment:
Code refactored and moved to client/routes/execution/helpers/get-time-elapsed-display.js & client/routes/execution/helpers/get-time-stamp-display.js